lib/src/room.dart

```diff
@@ -442,6 +451,25 @@ class KurentoRoom extends ChangeNotifier implements Room {
     }
   }
 
+  Future<void> _handleTriggers(String message) async {
+    Map<String, dynamic> json = jsonDecode(message);
+    _log.finest('Got following trigger: ${json['trigger']}');
+
+    // When a GPS Activated Offer is activated during a call, we get a 'SERVICE_ACCESS' message with the offer's details
+    if (json['trigger'] == 'SERVICE_ACCESS') {
+      Map<String, dynamic> value = json['value'] ?? [];
+      AccessOfferDetails accessOffer = AccessOfferDetails.fromJson(value['access']);
+      if (null == accessOffer.durationPerCall || accessOffer.durationPerCall! < 30) {
```

Comment:
I probably should have removed that. I was under the impression that we should receive a message when the offer only has 30 seconds left and I had put that there to avoid having both the "30 seconds" remaining and the initial time messages one after the other.

I'll remove it.